libs/shared/exceptions/exceptions/src/lib/exception.ts

```diff
@@ -51,35 +63,22 @@ export class BiosimulationsException extends Error {
       }
       error.source = source;
     }
+
     if (this.meta) {
       error.meta = this.meta;
     }
     return error;
   }
+
   getStatus(): number {
     return this.status;
   }
+
   getError(): ErrorObject {
     return this.errorObject;
   }
-
-  // TODO write test
-  static fromHTTP(exception: HttpException): BiosimulationsException {
```

Comment:
@bilalshaikh42 this isn't used anywhere. Please confirm this can be deleted.

Reply:
This was meant to be added to the error handlers for the HTTP exception filter to provide a consistent error object. If it is not used that would be the cause of #4017

Reply:
With the rest of the changes, this seems to be redundant now.

Reply:
That method was never hooked up to anything. The HTTP errors and BioSimulationsExceptions seem to be separate. I decided to standardize the titles of both by using the numeric codes to look up standardized titles.